components/activecampaign/activecampaign.app.mjs

```diff
@@ -145,35 +145,143 @@ export default {
         return this.listPipelineOptions(prevContext);
       },
     },
+    status: {
+      type: "integer",
+      label: "Status",
+      description: "Deal's status. Valid values:\n* `0` - Open\n* `1` - Won\n* `2` - Lost",
+      options: [
+        {
+          label: "Open",
+          value: 0,
+        },
+        {
+          label: "Won",
+          value: 1,
+        },
+        {
+          label: "Lost",
+          value: 2,
+        },
+      ],
+      optional: true,
+    },
+    fields: {
+      type: "string[]",
+      label: "Fields",
+      description: "Deal's custom field values [{customFieldId, fieldValue}]",
+      optional: true,
+    },
+    tags: {
+      type: "string[]",
+      label: "Tags",
+      description: "List of tags available",
+      optional: true,
+      async options() {
+        return this.listTagOptions();
+      },
+    },
+    contactTags: {
+      type: "string[]",
+      label: "Contact Tags",
+      description: "List of contact tags available",
+      async options({ contactId }) {
+        const { contactTags } = await this.getContactTags({
+          contactId,
+        });
+        return contactTags.map(({ id }) => ({
```

Comment:
since you're using `id` as `label` and `value`, you could return a simple string array